src/test/scala/net/kaliber/httpsRedirect/HttpsRedirectFilterSpecification.scala

```diff
@@ -1,41 +1,48 @@
 package net.kaliber.httpsRedirect
 
+import akka.actor.ActorSystem
+import akka.stream.ActorMaterializer
 import org.qirx.littlespec.Specification
+import play.api.Configuration
 import play.api.http.HeaderNames
 import play.api.mvc.AnyContentAsEmpty
 import play.api.mvc.Filter
 import play.api.mvc.RequestHeader
-import play.api.mvc.Result
 import play.api.mvc.Results.NoContent
-import play.api.mvc.Results.Redirect
 import play.api.test.FakeApplication
 import play.api.test.FakeHeaders
 import play.api.test.FakeRequest
 import play.api.test.Helpers
+import HttpsRedirectFilter._
+
 import scala.concurrent.Await
 import scala.concurrent.Future
 import scala.concurrent.duration._
 
 object HttpsRedirectFilterSpecification extends Specification {
 
-  import HttpsRedirectFilter._
-
   val host = "test.com"
   val sslPort = "1234"
 
-  "Tests using play configuration" - {
-    def createFilter(config: Map[String, Any] =
-      Map(
-        "httpsRedirectFilter.enabled" -> true,
-        "httpsRedirectFilter.sslPort" -> sslPort
-      )
+  def createFilter(config: Map[String, Any] =
+     Map(
+       "httpsRedirectFilter.enabled" -> DEFAULT_ENABLED,
+       "httpsRedirectFilter.sslPort" -> DEFAULT_SSL_PORT
+     )
```

Comment:
default config should be an empty map in order to make sure the correct code is triggered